modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -399,22 +403,11 @@ const Main = ({
 				const {error, file} = response.data;
 
 				disableSubmitButton(false);
-
-				if (error) {
-					configureErrorMessage(error.message);
-
-					setCurrentValue(null);
-
-					onChange(event, '{}');
-				}
-				else {
-					configureErrorMessage('');
-
-					setCurrentValue(JSON.stringify(file));
-
-					onChange(event, JSON.stringify(file));
-				}
-
+				handleGuestUploadFileChanged(
+					error ? error.message : '',
+					event,
+					error ? null : JSON.stringify(file)
+				);
```

Comment:
@carolmariaabb, let's avoid use ternary operators on the methods parameters in order to keep a good code readability. 😉 

One more thing, please keep the methods invocation separated by line breaks.